fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyAdapter.java

```diff
@@ -181,20 +183,57 @@ public boolean doesTableExist(String schemaName, String tableName) {
     }
 
     @Override
-    public void createSequence(String schemaName, String sequenceName, int cache) {
+    public void createSequence(String schemaName, String sequenceName, long startWith, int cache) {
         /* CREATE SEQUENCE fhir_sequence
          *     AS BIGINT
          *     START WITH 1
          *     CACHE 1000
          *     NO CYCLE;
-        */
+         */
```

Comment:
the above CREATE sequence seems out of place. - I suggest putting it where fhir_sequence is actually created or in a unit test

Reply:
It's just an example of the syntax we need to build. Comment will be made more generic.